app/assets/scss/_success-component.scss

```diff
@@ -2,4 +2,9 @@
   background-color: #f3f2f1;
   margin: 2rem 0;
   padding: 1rem 1.5rem;
-}
\ No newline at end of file
+}
+
+.list-item-no-bullet {
```

Comment:
remove this style, header should be outside <ul> tag so this would not be needed, read comment below about headers and lists.